DOCSP-45203 Single Field Indexes (#100)
mongodb/docs-ruby: source/indexes/single-field-index.txt

   This index is automatically created on the ``_id`` field when a new
   collection is created.

Sample Data
~~~~~~~~~~~

The examples in this guide use the ``movies`` collection in the
``sample_mflix`` database from the :atlas:`Atlas sample datasets
</sample-data>`. To access this collection from your {+language+} 
application, create a ``Mongo::Client`` object that connects to
an Atlas cluster and assign the following values to your ``database`` 
and ``collection``
variables: 

.. literalinclude:: /includes/indexes/single-field.rb
   :start-after: start-sample-data
   :end-before: end-sample-data
   :language: ruby
   :copyable:

To learn how to create a free MongoDB Atlas cluster and
load the sample datasets, see the :atlas:`Get Started with Atlas
</getting-started>` guide.

Create a Single Field Index
---------------------------

Use the ``create_one`` method to create a single
field index. The following example creates an index in ascending order on the
``title`` field:

.. literalinclude:: /includes/indexes/single-field.rb
   :start-after: start-index-single
   :end-before: end-index-single
   :language: ruby
   :copyable:

> Verify the Index's Creation

Reviewer: [s] Can potentially update this to [avoid using an article](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#style-and-structure) and be more concise:
```suggestion
Verify Index Creation
```

---------------------------

You can verify that the index was created by iterating through the ``indexes`` object

> to list the indexes in the collection. You should see an index for ``title`` in 
> the list, as shown in the following output:

Reviewer: [s] If you want to [avoid using](https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#use-modal-verbs-accurately) "should", can maybe edit to something like:
```suggestion
to list the indexes in the collection. The output includes an index for ``title`` in the list, as shown in the following example:
```

Author: There is a bit of uncertainty, depending on whether the the user is able to successfully create the index. Will leaves as is, if that makes sense to you!
